### DISCUSSION TITLE
Is there a way to format time for the hover plugin?

### DISCUSSION THREAD

**Discussion by AbhijithMergu** [katspaugh/wavesurfer.js#3364] [Q&A]
I'd like to display milliseconds on hover on the waveform. Is there a way to achieve it now? If not, can we have an issue raised for this?

**Comment by katspaugh** [comment]
There's no such option atm, but please feel free to make a pull request to add it.

It would be similar to the [`formatTimeCallback` option](https://github.com/katspaugh/wavesurfer.js/blob/main/src/plugins/timeline.ts#L29) in the timeline plugin.

The time formatting code is here: https://github.com/katspaugh/wavesurfer.js/blob/main/src/plugins/hover.ts#L96-L101
